views/demo.pug

```diff
@@ -0,0 +1,29 @@
+extends layout.pug
+
+block head
+	.navbar
+		ul
+			li
+				a.sdd-logo(href="/") Logo
+			li
+				a(href="#") Demo
+			li
+				a(href="/documentation") Documentation
+			li
+				a(href="/settings") App Settings
+		a(href="https://github.com/sahil505/StyleGuideDesigner", target="_blank"): img.icon(src="/img/github-icon-1.svg" alt="github")
+	hr
+block content
+	.container-fluid
+		.row
+			.col-sm-3.col-lg-2
+				nav.side-navbar.navbar-default.navbar-fixed-side
```

Comment:
I'm using classes names similar to bootstrap but not exactly bootstrap. I'll simplify the code.